profiler/build.gradle

```
@@ -0,0 +1,31 @@
+plugins {
+    id("com.github.johnrengelman.shadow") version "6.1.0"
+}
+
+apply from: "$rootDir/gradle/shadow.gradle"
+
+def relocatePackages = ext.relocatePackages
+
+dependencies {
+    compileOnly("io.opentelemetry:opentelemetry-sdk:${versions.opentelemetry}")
+    compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:${versions.opentelemetryAlpha}")
+    compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-spi:${versions.opentelemetryJavaagentAlpha}")
+    compileOnly("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api:${versions.opentelemetryJavaagentAlpha}")
+    compileOnly("io.opentelemetry:opentelemetry-semconv:1.0.1-alpha")
+    compileOnly deps.slf4j
+
+    annotationProcessor("com.google.auto.service:auto-service:1.0-rc7")
+    annotationProcessor("com.google.auto:auto-common:0.8")
+    implementation("com.google.auto.service:auto-service:1.0-rc7")
+    implementation("com.google.auto:auto-common:0.8")
```

You can use the `deps` helper map:
```suggestion
    annotationProcessor deps.autoservice
    implementation deps.autoservice
```